### PR TITLE
[WCM] Added information about the new date handling in the comparison constraints and Range

### PR DESCRIPTION
**Pull request by webmozart**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes (symfony/symfony#11673) |
| Applies to | 2.6+ |
| Fixed tickets | - |

### CODE REVIEW

**Comment by xabbuh** [comment]
@webmozart Will you be able to complete this or will you prefer if somebody takes this over?

File: reference/constraints/GreaterThan.rst [hunk 1]
```diff
@@ -83,6 +83,170 @@ If you want to ensure that the ``age`` of a ``Person`` class is greater than
...
+
+This constraint can be used to compare ``DateTime`` objects against any date
+string `accepted by the DateTime constructor`_. For example, you could check
+that a date must at least be the next day:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # src/OrderBundle/Resources/config/validation.yml
```
Comment (WouterJ) [inline reference/constraints/GreaterThan.rst]: it should use AppBundle now

**Comment by WouterJ** [comment]
Hi @webmozart. Do you have some time soon to finish this PR? If you don't, please say so as we can then assign it to someone else during the [doc sprint day](http://symfony.com/blog/symfony-docs-hack-day-needs-you-on-may-23rd) on May 23rd.

Thanks!

**Comment by javiereguiluz** [comment]
This PR has been finished in #5384

**Comment by xabbuh** [comment]
closing in favor of #5384

**Pull request closed without merging**